fatiando/gravmag/harvester.py

```diff
@@ -264,7 +265,10 @@ def harvest(data, seeds, mesh, compactness, threshold, report=False):
                       'misfit': data_misfit_value,
                       'regularizer': regularizing_function_value,
                       'accretions': number_of_accretions}
-
+    * restrict : list of str
+        Restricts seed growth in given directions. Possible directions are 
+        ``'above'``, ``'below'``, ``'front'``, ``'back'``, ``'left'`` and
+        ``'right'``.
```

Comment:
Please add something like: 

    You can pass in multiple directions as a list. Default is ``None`` for unrestricted growth.